cns/restserver/util.go

```diff
@@ -951,3 +999,40 @@ func (service *HTTPRestService) setResponse(w http.ResponseWriter, returnCode ty
 	serviceErr := service.Listener.Encode(w, &response)
 	logger.Response(service.Name, response, returnCode, serviceErr)
 }
+
+// ncList contains comma-separated list of unique NCs
+type ncList string
+
+// only add unique NC to ncList
+func (n *ncList) Add(nc string) {
```

Comment:
why did we change this implementation from set?

Reply:
This is because we wanted to keep nc to be added in order as discussed before but seems like they did change from their side. @ashvindeodhar do we keep this now?

Reply:
add more test cases to cover this.

Add 3 NCs, remove first, second and third one, and check if they are deleted in order

Reply:
test cases added